app/src/internal/res/values/environment_config.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="feeRelayerBaseUrl">https://fee-relayer.key.app/</string>
+
+    <string name="torusNetwork">cyan</string>
+    <string name="torusLogLevel">silent</string>
+    <string name="torusBaseUrl">https://metadata.tor.us</string>
```

Comment:
for firebase build we can use beardict
and network is testnet

Reply:
gotcha